integration-tests/utils/integration-utils.js

```diff
@@ -106,7 +106,7 @@ const svgText = ({ textContains }) => {
 }
 
 const navChild = text => {
-  return `//*[@role="tablist"]/*[contains(normalize-space(.),"${text}")]`
+  return `//*[@id="tab-navigation"]//a[contains(normalize-space(.),"${text}")]`
```

Comment:
I hate to poo-poo the idea, but we actually avoid using id's for the integration tests, on the basis that users don't target things by id, and we want to simulate user behavior. So, I'd rather target by aria role and then have to change it if we change the role in the future, rather than rely on something that's not exposed to the user.

Reply:
Ok, that makes sense